client/package.json

```diff
@@ -49,7 +48,8 @@
     "rimraf": "^3.0.0",
     "ts-jest": "^24.1.0",
     "typescript": "~3.7.5",
-    "vue-template-compiler": "^2.6.10"
+    "vue-template-compiler": "^2.6.10",
+    "vue2-hammer": "^2.1.2"
```

Comment:
Slight whoopsie :D  It's the `@types/vue2-hammer` dependency that has to go in the devDependencies as it provides nothing at runtime.  The `vue2-hammer` dependency has to be in the normal dependencies as it actually provides functionality.

Reply:
It looks like CI worked without it being in the standard dependencies, do you want me to revert it?

Reply:
Yes :D and move the @types to the devDepdencies